### PR TITLE
Team Antagonists will no longer be able to convert each other during Dynamic. (WIP)

### PR DESCRIPTION
**Pull request by Xoxeyos**
Darkspawn, Shadowlings, Shadowling Thralls, Clock Cultists, Blood Cultists, and Revolutionaries will not be able to convert each other during dynamic. Antagonists goals will no longer be as confusing. Antag Cap lowered for Traitor and BB by one each, testing to see if the error goes away.

:cl:  Xoxeyos
rscadd: Team conversion antagonists will not be able to convert each other, deconversion is required.
experimental: I'm a terrible coder, so I'm having the kinks worked out.
tweak: Antag cap lowered for Traitor and BB by one each.
/:cl:

### CODE REVIEW

**Comment by patpol4** [comment]
Blood brother is technically a team antag

**Comment by Xoxeyos** [comment]
> 
> 
> Blood brother is technically a team antag

Only revolutionaries will not be able to convert blood brothers, since I do believe heads can be blood brothers.

**Comment by boodaliboo** [comment]
if your deconverted from say blood cult and then became a thrall would that work as normal?

**Comment by Xoxeyos** [comment]
> 
> 
> if your deconverted from say blood cult and then became a thrall would that work as normal?

Yeah it should, you just shouldn't be able to be a blood cultist and a thrall.

**Comment by Xoxeyos** [comment]
Dynamic.json was changed so that it properly matched the roundstart traitor role for antag capacity, and for the weight.

**Comment by Xoxeyos** [comment]
Blood cult offerings are up next to be fixed, so hang tight. (In a round previously shadowling thralls could instantly be converted into the cult, while slings had to holy water them.)

File: config/dynamic.json [hunk 1]
```diff
@@ -24,13 +24,14 @@
...
+				1,
+				2,
 				2,
 				2,
 				2,
 				3,
 				3,
-				3,
-				3
+				4
```
Comment (JamieD1) [inline config/dynamic.json]:
```suggestion
```

**Review by JamieD1 — APPROVED** [review]

**Comment by Xoxeyos** [comment]
Now to fix cultist constructs becoming revolutionaries. I'm also going to assume they are obtaining other unnecessary roles, and I'll take care of that aswell.

**Comment by Xoxeyos** [comment]
This isn't stale, I've just had to crunch on finals, so don't worry, this is still being worked on, albeit at a slower pace.

**Comment by Xoxeyos** [comment]
Cultists aren't given objectives upon spawning.

**Comment by Xoxeyos** [comment]
Zombie revolutionaries bad, may as well try and prevent it now.

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**